ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -217,18 +221,21 @@ export const AccountListItem = ({
               {shortenAddress(toChecksumHexAddress(identity.address))}
             </Text>
           </Box>
-          <Text
-            variant={TextVariant.bodySm}
-            color={Color.textAlternative}
-            textAlign={TextAlign.End}
-            as="div"
-          >
-            <UserPreferencedCurrencyDisplay
-              ethNumberOfDecimals={MAXIMUM_CURRENCY_DECIMALS}
-              value={identity.balance}
-              type={SECONDARY}
-            />
-          </Text>
+          {showSecondaryBalance ? (
+            <Text
+              variant={TextVariant.bodySm}
+              color={Color.textAlternative}
```

Comment:
```suggestion
              color={TextColor.textAlternative}
```
I know this was there originally but if you can, would you mind making this change?

Reply:
Good catch!